crates/stark-backend/src/prover/quotient/evaluator.rs

```diff
@@ -24,10 +24,16 @@ impl<T> ViewPair<T> {
 
     pub fn get(&self, row_offset: usize, column_idx: usize) -> &T {
         match row_offset {
-            0 => &self.local[column_idx],
+            // SAFETY: all column indices have been checked to be in range already
+            0 => unsafe { self.local.get_unchecked(column_idx) },
```

Comment:
Hmm, `ViewPair` is a public type and `get` is public API. How do we know all this is unsafe in general? We probably should make the method unsafe and then callers can use this function if they have otherwise made the requisite checks. (Or am I missing something?)

Reply:
I changed to pub(crate) afterwards for this reason

Reply:
I think it's still easy to mess up and use `get` within this crate in an unsafe way. Isn't it better to mark the function as unsafe and then force the caller to think about it?